site/content/docs/5.2/components/dropdowns.md

```diff
@@ -774,6 +808,13 @@ Put a form within a dropdown menu, or make it into a dropdown menu, and use [mar
 
 Use `data-bs-offset` or `data-bs-reference` to change the location of the dropdown.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This variant should not be used because it does not respect the Orange Design System specifications.
```

Comment:
What about `data-bs-offset="0,0"` which is correct. Change the wording to explain that this rendering is not OK but the concept/feature can be used.

Reply:
OK => what about ?
> "This variant with a `data-bs-offset` attribute having values different than `"0,0"` should not be used because it does not respect the Orange Design System specifications.

> Please refer to the [Dropdown](https://system.design.orange.com/0c1af118d/p/910b9b-dropdown/b/04c480) guidelines on the Orange Design System website."

**Note** : The example "reference" which has no `data-bs-offset` attribute is less correct that the same with a  `data-bs-offset = "0,0"` => shall we correct the reference ?

Reply:
>Note : The example "reference" which has no data-bs-offset attribute is less correct that the same with a data-bs-offset = "0,0" => shall we correct the reference ?
Note no more relevant => the current design callout message can stay as is.